test/unit/connection/result/test_datetime_format_converter.js

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I think this file shouldn't be here? This is already tested in `datetime_format_converter_test`, so I assume it's a mistake.

Reply:
True. Fixed